### PR TITLE
fix react errors

### PR DESCRIPTION
**Pull request by AntonNazarenko**
fixed unique keys error, and all other console erros

### CODE REVIEW

File: src/ui/Input/Input.js [hunk 1]
```diff
@@ -26,13 +28,12 @@ export default class Input extends React.PureComponent {
...
 
     const { hidden } = this.state
-
     if (type === 'password') {
       return (
         <div className='hfui-input'>
           <input
             type={hidden ? 'password' : 'text'}
-            placeholder={placeholder}
+            placeholder={this.isPlaceholderValid(placeholder) ? placeholder : 'undefined'}
```
Comment (robertkowalski) [inline src/ui/Input/Input.js:36]: undefined as a string?

File: src/ui/Input/Input.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ export default class Input extends React.PureComponent {
           autoComplete={autocomplete}
           className={className}
           onChange={e => onChange(e.target.value)}
-          placeholder={placeholder}
+          placeholder={this.isPlaceholderValid(placeholder) ? placeholder : 'undefined'}
```
Comment (robertkowalski) [inline src/ui/Input/Input.js:62]: same here, you probably want to pass the object

Comment (AntonNazarenko) [reply src/ui/Input/Input.js]: @robertkowalski updated.

File: src/ui/Panel/index.js [hunk 3]
```diff
@@ -46,12 +46,12 @@ export default class Panel extends React.Component {
             <ul className='hfui-panel__header-tabs'>
               {tabs.map(tab => (
                 <li
-                  key={tab.props.htmlKey || tab.props.tabTitle}
```
Comment (robertkowalski) [inline src/ui/Panel/index.js:49]: we can remove `hmtlKey` from the props file: `Panel.props.js`

File: src/ui/Panel/index.js [hunk 4]
```diff
@@ -46,12 +46,12 @@ export default class Panel extends React.Component {
...
-                  className={ClassNames({ active: tab.props.tabTitle === selectedTab.props.tabTitle })}
+                  key={tab.props.tabtitle}
```
Comment (robertkowalski) [inline src/ui/Panel/index.js:49]: are you sure this works with the react elements passed in the TradeStatePanel? Thats the reason why I added htmlKey. Now I get this after login, our old error:

Screen Shot 2020-10-01 at 20 44 36

**Review by robertkowalski — CHANGES_REQUESTED** [review]
the old `[Object object]` issue on the Trading Terminal page is back. It was fixed by the `htmlKey` addition before

File: src/ui/Panel/index.js (hunk 4 above)
Comment (AntonNazarenko) [reply src/ui/Panel/index.js:49]: sry, just incorrect merge, I didn't want to delete this, thanks for noticing

Comment (AntonNazarenko) [reply src/ui/Panel/index.js]: updated

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**